README.md

```diff
@@ -212,6 +212,63 @@ From here on you can follow the instructions available at the following sections
 
 All done!
 
+Install ansible-openwisp2 on docker container for development
+-------------------------------------------------------------
+
+If you want to contribute to `ansible-openwisp2` you should test changes in
+replicated build's testing environment on your local machine.
+
+Proceed with these steps in your local machine to prepare your setup for development:
+
+**Step 1**: Clone `ansible-openwisp2`
+
+Clone repository by:
+
+    git clone https://github.com/<your_fork>/ansible-openwisp2.git
+
+**Step 2**: Install docker
+
+If you haven't installed docker yet, you need to install it (example for linux debian/ubuntu systems):
+
+    sudo apt-get install docker.io
+
+**Step 3**: Pull docker image
+
+Pull docker image with this command:
+
+    docker pull registry.gitlab.com/ninuxorg/docker/ansible-<distro>
+
+Substitute `<distro>` with one of these variants:
+
+* `ubuntu:18.04`
+* `ubuntu:16.04`
+* `debian:10`
+* `debian:9`
+
+**Step 4**: Run docker container
+
+To create and start docker container run this command:
+
+    docker run -d --volume=${PWD}/ansible-openwisp2:/etc/ansible/roles/role_under_test:rw --name <name> --privileged registry.gitlab.com/ninuxorg/docker/ansible-<distro> /lib/systemd/systemd
+
+Substitute `<name>` with desired name of container and `<distro>` with option you have chosen in **Step 3**.
+
+**Step 5** Install requirements on docker container
+
+When your container is running without any obstacles you need to install requirements:
+
+    docker exec -t <name> ansible-galaxy install -r /etc/ansible/roles/role_under_test/tests/requirements.yml
+
+**Step 5** Deploy **openwisp2** on docker container
+
+It's time deploy **openwisp2** with:
+
+    docker exec -t <name> ansible-playbook /etc/ansible/roles/role_under_test/tests/test.yml
+
+If you won't get any error messages it means that you **successfully deployed openwisp2**.
```

Comment:
`If you don't get any error message it means that the tests ran successfully without errors.`